contracts/foundry-tests/ChainlinkMultiOracle.t.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.6;
+
+import "@yield-protocol/utils-v2/contracts/access/AccessControl.sol";
+import "../oracles/chainlink/ChainlinkMultiOracle.sol";
+import "../mocks/DAIMock.sol";
+import "../mocks/USDCMock.sol";
+import "../mocks/WETH9Mock.sol";
+import "../mocks/oracles/OracleMock.sol";
+import "../mocks/oracles/chainlink/ChainlinkAggregatorV3Mock.sol";
+import "./utils/Test.sol";
+import "./utils/TestConstants.sol";
+
+contract ChainlinkMultiOracleTest is Test, TestConstants, AccessControl {
+    OracleMock public oracleMock;
+    DAIMock public dai;
+    USDCMock public usdc;
+    WETH9Mock public weth;
+    ChainlinkMultiOracle public chainlinkMultiOracle;
+    ChainlinkAggregatorV3Mock public daiEthAggregator; 
+    ChainlinkAggregatorV3Mock public usdcEthAggregator;
+
+    bytes32 public mockBytes32 = 0x0000000000000000000000000000000000000000000000000000000000000001;
+    bytes6 public mockBytes6 = 0x000000000001;
+    uint256 public oneUSDC = WAD / 1000000000000;
+
+    function setUp() public {
+        oracleMock = new OracleMock();
+        dai = new DAIMock();
+        usdc = new USDCMock();
+        weth = new WETH9Mock();
+        daiEthAggregator = new ChainlinkAggregatorV3Mock();
+        usdcEthAggregator = new ChainlinkAggregatorV3Mock();
+        chainlinkMultiOracle = new ChainlinkMultiOracle();
+        chainlinkMultiOracle.grantRole(0xef532f2e, address(this));
+        chainlinkMultiOracle.setSource(DAI, dai, ETH, weth, address(daiEthAggregator));
+        chainlinkMultiOracle.setSource(USDC, usdc, ETH, weth, address(usdcEthAggregator));
+        vm.warp(uint256(mockBytes32));
+        daiEthAggregator.set(WAD / 2500);
+        usdcEthAggregator.set(WAD / 2500);
+    }
+
+    function testGetSpotPrice() public {
```

Comment:
Actually, this doesn't test the spotPrice. This should be `testGetConversion`. We are testing the value of `base` in `quote` terms using `get` (we could be using `peek` as well).

Below it's the same in all tests. We test an asset conversion, using `get` or `peek`.

[Related reading](https://hackernoon.com/getting-prices-right)